Go Client/Go Rules.cpp

```diff
@@ -1,93 +1,93 @@
 #include "Go Rules.h"
 
 Rules::Rules(GameState &goBoard) :
-	currentBoard(goBoard.GoBoard())
+	m_currentBoard(goBoard.GoBoard())
 {
 }
 
 void Rules::DoRules(stone stoneChecking, GameState &goBoard)
 {
-	currentBoard = goBoard.GoBoard();
-	for (uint8_t x = 0; x < currentBoard.size(); ++x)
-	{
-		for (uint8_t y = 0; y < currentBoard.size(); ++y)
-		{
-			CheckRules(x, y, goBoard, stoneChecking);
-		}
-	}
+   m_currentBoard = goBoard.GoBoard();
+   for (uint8_t x = 0; x < m_currentBoard.size(); ++x)
+   {
+      for (uint8_t y = 0; y < m_currentBoard.size(); ++y)
+      {
+         CheckRules(x, y, goBoard, stoneChecking);
+      }
+   }
 }
 
-void Rules::CheckRules(uint8_t x, uint8_t y, GameState &goBoard, stone StoneChecking)
+void Rules::CheckRules(uint8_t x, uint8_t y, GameState &goBoard, stone stoneChecking)
```

Comment:
not very clear what the purpose of stoneChecking is from it's name. colourToCheck?
also the function isn't just checking the rules, it's also enforcing them, so maybe that should be renamed